### PR TITLE
allows change for bots game or stream status

### PR DESCRIPTION
**Pull request by chrislennon**
Refers to #234 

May want to allow for pass in of a url to the `!stream` so it will be clickable within the bot, currently its only going to twitch homepage.

### CODE REVIEW

**Comment by otter-computer** [comment]
This adds 2 commands that control the status message of the bot (e.g. "Playing XXX"). 
`!status` will change the game it's playing. 
`!stream` will change the message and add a link to the [gaymers discord twitch](https://www.twitch.tv/gaymersdiscord) while turning the icon purple, like any normal member that is currently streaming on twitch.

**Review by brandon-a — APPROVED** [review]

File: commands/status.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+const splitargs = require('splitargs');
+
+module.exports = {
+  usage: 'Message to display,',
+  description: 'ADMIN ONLY: Set the status of the bot.',
+  allowDM: false,
+  requireRoles: ['Admin', 'Moderator'],
+  process: (bot, message) => {
+
+    let status = splitargs(message.content);
```
Comment (jmariondev) [inline commands/status.js]: `const` :wank:

File: commands/status.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    status.shift();
+
+    let statusMessage = status.join(' ');
```
Comment (jmariondev) [inline commands/status.js]: `const` :wank:

File: commands/stream.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+const splitargs = require('splitargs');
+
+module.exports = {
+  usage: 'Message to display as a stream.',
+  description: 'ADMIN ONLY: Set the stream status of the bot.',
+  allowDM: false,
+  requireRoles: ['Admin', 'Moderator'],
+  process: (bot, message) => {
+
+    let status = splitargs(message.content);
```
Comment (jmariondev) [inline commands/stream.js]: `const` :wank:

File: commands/stream.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    status.shift();
+
+    let statusMessage = status.join(' ');
```
Comment (jmariondev) [inline commands/stream.js]: `const` :wank:

File: commands/stream.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+    if (status.length > 0){
+        bot.user.setGame(statusMessage,'https://www.twitch.tv/gaymersdiscord');
```
Comment (jmariondev) [inline commands/stream.js]: Cosmetic: `s/,/, /`

**Review by jmariondev — CHANGES_REQUESTED** [review]
Couple nitpicky things, else LGTM!

File: commands/stream.js (hunk 3 above)
Comment (otter-computer) [reply commands/stream.js]: LOL JAVASCRIPT ALLOWING MODIFICATION ON A CONST

Comment (jmariondev) [reply commands/stream.js]: Inner mutability 😉

Comment (otter-computer) [reply commands/stream.js]: My brackets are the only part of me that's certain.

**Pull request merged**